ildcp-core/src/test/java/org/interledger/ildcp/InterledgerShampooPacket.java

```diff
@@ -0,0 +1,18 @@
+package org.interledger.ildcp;
+
+import org.interledger.core.Immutable;
+import org.interledger.core.InterledgerResponsePacket;
+
+@Immutable
+public interface InterledgerShampooPacket extends InterledgerResponsePacket {
```

Comment:
Let’s add some Javadoc to explain what this is?

Reply:
Added; it's only in the `test` package for testing fall through cases on `isAssignableFrom` checks.